jobrunner/log_utils.py

```diff
@@ -103,7 +105,8 @@ def __init__(self, status_codes_to_ignore):
         self.status_codes_to_ignore = set(status_codes_to_ignore)
 
     def filter(self, record):
-        return record.status_code not in self.status_codes_to_ignore
+        status = getattr(record, "status", None)
```

Comment:
Shouldn't this be `status_code`?

Reply:
It's correct but confusing - status is always be available.

I think I'll change it, as it's not clear.